### PR TITLE
Add BBRA.eth space (ENS)

### PR DESCRIPTION
**Pull request by boobank**
**If you want to create a space follow this guide**

https://docs.snapshot.page/guides/create-a-space

### CODE REVIEW

File: skins/bbra.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.bbra{
+  --primary-color: rgb(173 48 48);
```
Comment (bonustrack) [inline skins/bbra.scss:2]:
```suggestion
  --primary-color: rgb(173, 48, 48);
```

File: skins/bbra.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  --heading-color: #000000;
+  --border-color: rgb(81, 78, 51);
+  --header-bg: #FBDCEF;
+  --block-bg: #FBDCEF;
+
+  #topnav {
+      border-bottom: 3px solid black !important;
+      box-shadow: rgb(192, 143, 189) -1px 1px 10px 2px;
+  }
+  .border-md {
```
Comment (bonustrack) [inline skins/bbra.scss:15]: You shouldn't change the style of the css classes, #topnav is the only exception.

Comment (boobank) [reply skins/bbra.scss:15]: Shouldn't or mustn't? Happy to take the chances if you guys allow!

**Review by gokaiorg — CHANGES_REQUESTED** [review]
Please use only theme variables. No extra custom styles are allowed.

**Comment by gokaiorg** [comment]
There is a conflict in your PR, can you resolve it?

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**